vividus-extension-selenium/src/test/java/org/vividus/bdd/converter/ui/AbstractParametersToSequenceActionConverterTests.java

```diff
@@ -34,21 +35,21 @@
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
-import org.openqa.selenium.Point;
-import org.vividus.bdd.converter.PointConverter;
-import org.vividus.bdd.converter.ui.StringToLocatorConverter;
-import org.vividus.bdd.steps.ui.web.model.SequenceAction;
-import org.vividus.bdd.steps.ui.web.model.SequenceActionType;
+import org.openqa.selenium.WebElement;
+import org.vividus.bdd.steps.ui.model.ISequenceActionType;
+import org.vividus.bdd.steps.ui.model.SequenceAction;
 import org.vividus.ui.action.search.Locator;
 
 @ExtendWith(MockitoExtension.class)
-class ParametersToSequenceActionConverterTests
+class AbstractParametersToSequenceActionConverterTests
 {
     private static final String TEXT = "text";
 
-    @Mock private StringToLocatorConverter stringToLocatorConverter;
-    @Mock private PointConverter pointConverter;
-    @InjectMocks private ParametersToSequenceActionConverter converter;
+    @Mock
+    private StringToLocatorConverter stringToLocatorConverter;
+
+    @InjectMocks
+    private TestParametersToSequenceActionConverter converter;
```

Comment:
please follow mock one line definition as it was before, be slowly moving to apply this style everywhere :)